### ISSUE TITLE
Relationships between trophic level biomass

### ISSUE THREAD

**Issue by anders-kolstad** [NINAnor/ecosystemCondition#172]
There is necessarily some uncertainty around the relationship that we expect between trophic levels under the reference condition. These things are hard to know. 

For plant-herbivore relationships we use a global relationship from [Fløygaard et al 2022](https://besjournals.onlinelibrary.wiley.com/doi/10.1111/1365-2664.14047). 
This model has an R $^{2}$ or only 0.04. Although the plant-herbivore relationship is strong in Africa and in N-America, the model fit is highly reduced when including Europe and Asia with impoverished herbivore communities. Could we use the African model? But what then about seasonal ecosystems? In our examples from Norway we see surprisingly little evidence of overgrazing in N-Norway (where we know semi-domestic reindeer herds are quite big). Could this be due to seasonal ecosystem having a reltively high summer production, but the animals must survive during the winter as well, when grazing/browsing has a more detrimental effect on vegetation?

Predator-prey relationships ([Hatton et al 2015](https://www.science.org/doi/10.1126/science.aac6284) are perhaps more robust? Citation: "The striking similarities that are observed across different kinds of systems imply a process that does not depend on system details. The regularity of many of these relations allows large-scale predictions and suggests high-level organization."

**Comment by JamesDMSpeed** [comment]
Good point - There is indeed a lot of unaccounted variation in herbivore biomass from NPP at 1km from the global relationship. The global relationship at 100km has a higher coefficient of determination (but still only 0.10). The NPP scale issue also reflects the comment in issue  #173 - choosing a larger spatial scale links better to the herbivore population and its range, where a smaller spatial scale (as we have used so far) emphasizes the potential impact of the herbivores on the vegetation at a single site. 

The African >20kg herbivores has the highest coefficient of determination of 0.63. But I am not certain that we can justify using an African model for Norwegian herbivores. You raise the issue of seasonality but I think that the functional variation in herbivores is far more important - as Africa still has mega-herbivores while Norway does not.  

Exploring the implications of using some of the other models published in Fløjgaard et al 2022 for our predictions would be an interesting follow up.
